src/emu/render.cpp

```diff
@@ -1574,38 +1574,80 @@ void render_target::update_layer_config()
 
 void render_target::load_layout_files(const internal_layout *layoutfile, bool singlefile)
 {
-	bool have_default = false;
+	bool have_default  = false;
+	bool have_artwork  = false;
+	bool have_override = false;
 
 	// if there's an explicit file, load that first
 	const char *basename = m_manager.machine().basename();
 	if (layoutfile)
-		have_default |= load_layout_file(basename, layoutfile);
+		have_artwork |= load_layout_file(basename, layoutfile);
 
 	// if we're only loading this file, we know our final result
 	if (singlefile)
 		return;
 
-	// try to load a file based on the driver name
+	// if override_artwork defined, load that and skip artwork other than default
+	if (m_manager.machine().options().override_artwork())
+	{
+		if (load_layout_file(m_manager.machine().options().override_artwork(), m_manager.machine().options().override_artwork()))
+			have_override |= true;
+		if (load_layout_file(m_manager.machine().options().override_artwork(), "default"))
```

Comment:
Is it intentional that you're still attempting to load `"default"` even if call above succeeded?  Should there be an `else` at the beginning of this line?

Reply:
It was intentional to make both available to the user (with the specific as default), but if you prefer I can add the else. :)

Reply:
Well, an artwork pack should only contain one `.lay` file, and the current behaviour is to look for, e.g. `artwork/superx/superx.lay` and if that fails to try `artwork/superx/default.lay` (i.e. it won't consider `default.lay` if it can load `superx.lay`).  It should be the same for the override option - put the `else` in.